src/cosmosdb-preview/azext_cosmosdb_preview/custom.py

```diff
@@ -85,52 +86,57 @@ def cli_cosmosdb_mongocluster_firewall_rule_create(client,
                                                    end_ip_address):
 
     '''Creates an Azure Cosmos DB Mongo Cluster Firewall rule'''
-        
-    firewall_rule =  FirewallRule(start_ip_address=start_ip_address, end_ip_address=end_ip_address)
-    
+
+    firewall_rule = FirewallRule(start_ip_address=start_ip_address, end_ip_address=end_ip_address)
+
     return client.begin_create_or_update_firewall_rule(resource_group_name, cluster_name, rule_name, firewall_rule)
 
+
 def cli_cosmosdb_mongocluster_firewall_rule_update(client,
-                                    resource_group_name,
-                                    cluster_name,
-                                    rule_name,
-                                    start_ip_address,
-                                    end_ip_address):
+                                                   resource_group_name,
+                                                   cluster_name,
+                                                   rule_name,
+                                                   start_ip_address,
+                                                   end_ip_address):
 
     '''Creates an Azure Cosmos DB Mongo Cluster Firewall rule'''
-    
+
     mongo_cluster_firewallRule = client.get_firewall_rule(resource_group_name, cluster_name, rule_name)
 
     if start_ip_address is None:
         start_ip_address = mongo_cluster_firewallRule.startIpAddress
-    
+
     if end_ip_address is None:
         firewall_rule_end_ip_address = mongo_cluster_firewallRule.endIpAddress
-    
-    firewall_rule =  FirewallRule(start_ip_address=start_ip_address, end_ip_address=end_ip_address)
-    
+
+    firewall_rule = FirewallRule(start_ip_address=start_ip_address, end_ip_address=end_ip_address)
+
     return client.begin_create_or_update_firewall_rule(resource_group_name, cluster_name, rule_name, firewall_rule)
 
+
 def cli_cosmosdb_mongocluster_firewall_rule_list(client, resource_group_name, cluster_name):
 
     """List Azure CosmosDB Mongo Cluster Firewall Rule."""
 
     return client.list_firewall_rules(resource_group_name, cluster_name)
 
+
 def cli_cosmosdb_mongocluster_firewall_rule_get(client, resource_group_name, cluster_name, rule_name):
 
     """Gets Azure CosmosDB Mongo Cluster Firewall rule"""
 
     return client.get_firewall_rule(resource_group_name, cluster_name, rule_name)
 
+
 def cli_cosmosdb_mongocluster_firewall_rule_delete(client, resource_group_name, cluster_name, rule_name):
 
     """Delete Azure CosmosDB Mongo Cluster Firewall Rule"""
 
     return client.begin_delete_firewall_rule(resource_group_name, cluster_name, rule_name)
 
+
 def cli_cosmosdb_mongocluster_create(client,
-                                    resource_group_name,
+                                     resource_group_name,
```

Comment:
nit: remove extra space

Reply:
this is a style fix from the azdev style command